test/data/jwt_keys/private_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Would prefer this to be stored in a github secret. I know it is just for test.

Reply:
On a later discussion we decided to leave these files alive, but It is not a problem to be deleted after the last changes. @blagoev , @nirinchev shall I delete them?

Reply:
If these are not needed lets remove them. Right now they have not polluted the main branch so if we delete them they will not get into the master history